homepage/src/pages/resource.jsx

```diff
@@ -0,0 +1,97 @@
+import Banner from '../components/banner';
+import TwoColumns from '../components/twoColumns';
+import ThreeColumns from '../components/threeColumns';
+import ImageAndText from '../components/imageAndText';
+
+// ? Not used in index.md, just for Generic part
+import Section from '../components/section';
+import Headline from '../components/headline';
+
+import { fetchPage } from '../lib/fetchPage';
+
+/**
+ *
+ * @type {import('next').GetStaticProps}
+ */
+export const getStaticProps = async ({ locale }) => {
+  const page = fetchPage(locale, 'index');
```

Comment:
I've created the basic resource page. We can replace the `index` w/ `resource` safely.

Reply:
ok!

Reply:
ok!